server/php-master.cpp

```diff
@@ -1239,17 +1138,10 @@ int php_master_wakeup(connection *c) {
     c->status = conn_expect_query;
   }
 
-  pmm_data *D = PMM_DATA (c);
-  bool full_flag = D->full_flag;
-  int worker_pid = D->worker_pid;
-
   update_workers();
 
-  std::string res = php_master_prepare_stats(full_flag, worker_pid);
-  return_one_key_val(c, (char *)res.c_str(), (int)res.size());
-  if (D->need_end) {
-    write_out(&c->Out, "END\r\n", 5);
-  }
+  std::string res = php_master_prepare_stats();
+  return_one_key_val(c, res.c_str(), static_cast<int>(res.size()));
 
```

Comment:
END is set only if `pending_queries != 0`, since the pending_queries value is always zero (because no pipe communication), there is no need to set END at this place anymore. It is set now only from `php_master_get_end()`.